src/schema/v2/conversation/conversations.ts

```diff
@@ -18,22 +48,65 @@ const Conversations: GraphQLFieldConfig<void, ResolverContext> = {
     },
   }).connectionType,
   description: "Conversations, usually between a user and partner.",
-  args: pageable(),
-  resolve: (_root, options, { conversationsLoader, userID }) => {
-    if (!conversationsLoader) return null
-    const { page, size, offset } = convertConnectionArgsToGravityArgs(options)
+  args: pageable({
+    dismissed: {
+      type: GraphQLBoolean,
+    },
+    hasMessage: {
+      type: GraphQLBoolean,
+    },
+    hasReply: {
+      type: GraphQLBoolean,
+    },
+    partnerId: {
+      type: GraphQLString,
+    },
+    type: {
+      type: ConversationsTypeEnum,
```

Comment:
#superminor this feels like rather than an implicit default of 'User' type (due to if/else construct and backwards compatibility), maybe this should actually be non-null but with a default of User (i _think_ that works w/ non-null w/o breaking clients if a default is specified)?

Reply:
I did have this set as a default via an Enum `defaultValue` but using enum default values is weird! I'll ping you in a bit and will show you what i was running into.